tests/Unit/Evolution/Actions/Test_ComputeVolumeFluxes.cpp

```diff
@@ -73,30 +78,46 @@ struct component {
 };
 
 struct Metavariables {
+  static constexpr size_t volume_dim = dim;
   using component_list = tmpl::list<component<Metavariables>>;
   using system = System;
   enum class Phase { Initialization, Testing, Exit };
 };
-}  // namespace
 
-SPECTRE_TEST_CASE("Unit.Evolution.ComputeVolumeFluxes",
-                  "[Unit][Evolution][Actions]") {
+template <bool UseMovingMesh>
+void test() noexcept {
   using MockRuntimeSystem = ActionTesting::MockRuntimeSystem<Metavariables>;
-
   const ElementId<dim> self_id(1);
-
-  using simple_tags = db::AddSimpleTags<Var1, Var2, flux_tag>;
   MockRuntimeSystem runner{{}};
+
+  boost::optional<tnsr::I<DataVector, dim, Frame::Inertial>> mesh_velocity{};
+  if (UseMovingMesh) {
+    mesh_velocity =
+        tnsr::I<DataVector, dim, Frame::Inertial>{{{{1.2}, {-1.4}}}};
+  }
   ActionTesting::emplace_component_and_initialize<component<Metavariables>>(
       &runner, self_id,
-      {db::item_type<Var1>{{{3.}}}, db::item_type<Var2>{{{7., 12.}}},
-       db::item_type<flux_tag>{{{-100.}}}});
+      {Variables<tmpl::list<Var1>>{1, 3.},
+       tnsr::I<DataVector, dim, Frame::Inertial>{{{{7.}, {12.}}}},
+       db::item_type<flux_tag>{1, -100.}, mesh_velocity});
   ActionTesting::set_phase(make_not_null(&runner),
                            Metavariables::Phase::Testing);
   runner.next_action<component<Metavariables>>(self_id);
 
-  auto& box = ActionTesting::get_databox<component<Metavariables>, simple_tags>(
-      runner, self_id);
-  CHECK(get<0>(db::get<flux_tag>(box)) == -15.);
-  CHECK(get<1>(db::get<flux_tag>(box)) == 57.);
+  tnsr::I<DataVector, dim, Frame::Inertial> expected_flux{{{{-15.}, {57.}}}};
+  if (UseMovingMesh) {
+    get<0>(expected_flux) -= get<0>(*mesh_velocity) * 3.;
+    get<1>(expected_flux) -= get<1>(*mesh_velocity) * 3.;
```

Comment:
This 3 feels a bit like a magic number. Can you replace it here (and where it sourced the variable above) with a `const double var1_value = 3.;` or similar?